Doc/whatsnew/3.14.rst

```diff
@@ -420,6 +420,11 @@ asyncio
 Deprecated
 ==========
 
+* :mod:`argparse`:
+  Passing *prefix_chars* to :func:`argparse.add_argument_group`
```

Comment:
```suggestion
  Passing undocumented keyword argument *prefix_chars* to :meth:`~argparse.ArgumentParser.add_argument_group`
```

And perhaps this should be added to `Doc/deprecations/pending-removal-in-future.rst`.